src/main/java/com/snowflake/snowpark_java/Functions.java

```diff
@@ -372,6 +372,17 @@ public static Column sum(Column col) {
     return new Column(com.snowflake.snowpark.functions.sum(col.toScalaColumn()));
   }
 
+  /**
+   * Returns the sum of non-NULL records in a group. You can use the DISTINCT keyword to compute the
+   * sum of unique non-null values. If all records inside a group are NULL, the function returns
+   * NULL.
+   *
+   * @since 0.9.0
+   * @param str The input string
```

Comment:
1. please change `@since 0.9.0` to `@since 1.12.0`
2. change `@param str The input string` to `@param str The input column name`. Actually, we should change the name `str` to `colName`.
3. how the keyword `DISTINCT` works in this function? It maybe not work in this function, so either remove this part from doc, or add related test.

Reply:
Thanks @sfc-gh-bli, like you said, the `DISTINCT` keyword doesn't work in this function so I removed that part from the comments.